exchanges/stream/websocket.go

```diff
@@ -232,18 +232,11 @@ func (w *Websocket) SetupNewConnection(c ConnectionSetup) error {
 	}
 
 	newConn := &WebsocketConnection{
-		ExchangeName:      w.exchangeName,
-		URL:               connectionURL,
-		ProxyURL:          w.GetProxyAddress(),
-		Verbose:           w.verbose,
-		ResponseMaxLimit:  c.ResponseMaxLimit,
-		Traffic:           w.TrafficAlert,
```

Comment:
I'm leaving this as just a comment for now. I think a TrafficAlert from parent only seems like a problematic decision for next steps. It feels like things like Traffic and Wg should be moving from `w` to the individual connection instead of a shared point